src/BaseGraph.ts

```diff
@@ -238,6 +238,36 @@ export abstract class BaseGraph {
     return people ? people.value : null;
   }
 
+  /**
+   * async promise, returns all channels associated with the user logged in
+   *
+   * @returns {Promise<MicrosoftGraph.Channel[]>}
+   * @memberof Graph
+   */
+  public async picker_GetAllMyTeams(): Promise<any> {
+    const teams = await this.client
+      .api('/me/joinedTeams')
+      .middlewareOptions(prepScopes('User.Read.All'))
+      .get();
+
+    return teams ? teams.value : null;
+  }
+
+  /**
+   * async promise, returns all channels associated with the user logged in
+   *
+   * @returns {Promise<MicrosoftGraph.Channel[]>}
+   * @memberof Graph
+   */
+  public async picker_GetChannelsFromTeam(teamId: string): Promise<any> {
```

Comment:
Should be `getChannelsForTeam`. These methods should also be type safe by returning the appropriate type. Should this be `Promise<MicrosoftGraph.Channel[]>`? Same for other methods